docs/docsite/rst/user_guide/playbooks_advanced_syntax.rst

```diff
@@ -50,6 +50,50 @@ For complex variables such as hashes or arrays, ``!unsafe`` should be used on th
         unsafe_key: !unsafe 'unsafe value'
 
 
+Using nested variables
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to refer to the same object from different parts of playbook, you need to use anchors and aliases.
+Anchors are denoted by ``&`` indicator while aliases are denoted by ``*``
+
+In a playbook, this may look like::
+
+    ---
+    ...
+    vars:
+        app1:
+            jvm: &jvm_opts
+                opts: '-Xms1G -Xmx2G'
+                port: 1000
+                path: /usr/lib/app1
+        app2:
+            jvm:
+                <<: *jvm_opts
+                path: /usr/lib/app2
+    ...
+
+Here, 'app1' and 'app2' variables shares 'opts' and 'port' information so you can anchor that
+information using ``&jvm_opts`` and re-use this information by indicating ``<<: *jvm_opts``.
+
+Another example can be::
+
+    ---
+    - name: Using values nested inside dictionary
+      hosts: localhost
+      vars:
+        webapp:
+            app_version: &my_version 1.0
+            custom_version:
+                - "ToDo_App"
+                - *my_version
+      tasks:
+      - name: Using Anchor value
+        debug:
+            msg: "{{ webapp.custom_version | join('-') }}"
+
+Here, you can anchor 'app_version' value using ``&my_version`` and re-use later as ``*my_version``.
+This way you can access nested values inside dictionaries.
```

Comment:
Confused here. I don't see *my_version used at all here? Did you mean to use msg: "{ *my_version | join('-') }} here?